components/story-element.js

```diff
@@ -80,6 +94,10 @@ class StoryElement extends React.Component {
       componentWillUnmount.call(this);
   }
 
+  componentWillReceiveProps(){
+    trackStoryElementView(this.props);
```

Comment:
dei, this doesn't work. It will just keep firing over and over

```javascript
componentWillRecieveProps(nextProps) {
  if(nextProps.inView && !this.state.onViewTriggered) {
    this.setState({onViewTriggered: true});
    trackStoryElementView(nextProps);
  } 
}
```